feconf.py

```diff
@@ -681,7 +681,7 @@ def get_empty_ratings():
 COLLECTION_EDITOR_URL_PREFIX = '/collection_editor/create'
 COLLECTION_URL_PREFIX = '/collection'
 COMMUNITY_OPPORTUNITIES_DATA_URL = '/opportunitiessummaryhandler'
```

Comment:
Ditto here and below -- variable name should use CONTRIBUTOR?

Reply:
Just noting that this one isn't fixed yet...

You might want to self-review the diff of the PR to make sure there aren't any inadvertent errors.

Reply:
I believe you were looking at the outdated view?